### ISSUE TITLE
Mailoney no logs - custom domain for some sensor

### ISSUE THREAD

**Issue by Fosocles** [dtag-dev-sec/tpotce#596]
Hi. I am currently using TPot behind a firewall with VIPs to separate some of the services and give it a bit more credibility. 
I might be mistaken, but Mailoney works as an open-relay stmp server (correct me if I'm wrong). When I run `hptest.sh` from private IP to TPot's private IP, I get logs in Mailoney (and Honeytrap), but when I test TPot from an external IP, Mailoney does not register any logs (neither does Honeytrap). 
After some brainstorming, I came to the conclusion that a smtp relay should to be registered in a DNS record. I'm willing to buy a custom domain and if I do so, I want to know if it is a good ideea to pair Snare (and consequently the Wordpress page it's currently serving) with this domain and serve dorks to increase the attack surface.

Tl;dr: 
1. Do I need a domain for Mailoney to get logs from outside the firewall? 
2. Is it a good idea to get a custom domain (for Mailoney and Snare) and serve dorks in Snare?

**Comment by t3chn0m4g3** [comment]
This depends on so many different factors it would be careless of me to answer your question in that regard based on the information given.

**Comment by Fosocles** [comment]
Ok, I can understand.
But, can you tell me at least if you get logs on Mailoney and Honeytrap?

**Comment by t3chn0m4g3** [comment]
[Yes, we do.](https://sicherheitstacho.eu/start/main)

**Comment by Fosocles** [comment]
Very strange. From what you can tell me, are your instances behind firewalls or something?

**Comment by t3chn0m4g3** [comment]
I am running my privately owned honeypots behind a firewall and they are set up according to the README with port forwarding and NAT.

**Comment by Fosocles** [comment]
@t3chn0m4g3 Funny thing: I had an old logstash.conf volume mounted and it had an outdated Mailoney filter (hence the missing logs). It's all good now.
